services/agora/.eslintrc.cjs

```diff
@@ -77,7 +77,7 @@ module.exports = {
         // in plain CommonJS modules, you can't use `import foo = require('foo')` to pass this rule, so it has to be disabled
         "@typescript-eslint/no-var-requires": "off",
 
-        "@vue/multi-word-component-names": "off",
+        "vue/multi-word-component-names": 0,
```

Comment:
We had migrated to a "file based" routing which uses "index.vue" within the routes folder. The idea is that the routes will follow the folder structure. Therefore, if I leave this setting on, Quasar will treat it as an error because "index" is a single word.

`Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"on"').`

Off is actually an invalid value.